README.md

```diff
@@ -31,18 +31,26 @@ No required modules.
 ### Required software
 - [GNU make](https://man7.org/linux/man-pages/man1/make.1.html)
 - [Docker](https://www.docker.com/)
+- [Consul](https://releases.hashicorp.com/consul/)
+- [psql CLI](https://www.postgresqltutorial.com/install-postgresql/)
 
 ## Usage
 The following command will run a standalone instance of postgres found in the [example](/example) folder.
 
-```text
+```sh
 make test
 ```
 
 ### Verifying setup
-You can verify that postgres is running by checking the connection. This can be done using the `consul` docker image to set up a proxy. Check out the [required software](#required-software) section.
-```shell script
-make proxy-to-postgres
+You can verify that Postgres is running by checking the connection. The following command uses Consul ([required software](#required-software)) to set up a proxy.
+```sh
+make proxy
+```
+
+Further, you can verify the connection by connecting with the psql CLI ([required software](#required-software)) using the command bellow.
+You can find the `username` and `password` in the [Vault UI (localhost:8200)](http://localhost:8200/).
+```sh
+psql "dbname=metastore host=127.0.0.1 user=<username> password=<password> port=9000 sslmode=disable"
```

Comment:
Could you change the binding port for the proxy to `5432` or smth else than `9000`?

Reply:
Sure